hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerBalancerStatusSubcommand.java

```diff
@@ -33,13 +45,132 @@
     versionProvider = HddsVersionProvider.class)
 public class ContainerBalancerStatusSubcommand extends ScmSubcommand {
 
+  @CommandLine.Option(names = {"-v", "--verbose"},
+          description = "Verbose output. Show current iteration info.")
+  private boolean verbose;
+
+  @CommandLine.Option(names = {"-vh", "--verbose-with-history"},
+      description = "More verbose output. Show current and history iteration info.")
+  private boolean verboseWithHistory;
```

Comment:
This option would be better as `-H, --history`, and handled as an extra for when verbose output is enabled by `--verbose`.  It can be ignored when `verbose==false`.  This simplifies logic both for users and for implementation (no need for `verbose || verboseWithHistory`).